### ISSUE TITLE
Extend DFT information

### ISSUE THREAD

**Issue by ndaelman-hu** [FAIRmat-NFDI/nomad-simulations#13]
Currently, only a handful of _hybrid_ DFs currently know their default exchange factor mixing.
This coverage should be extended, preferably without hard-coding the defaults. 

Alternative solutions may include using the libxc Python module, or producing an artifact from our own graph database.
In case of the latter, we should also consider whether registering the default is appropriate for the normalizer, or should happen at a higher level.

**Comment by JosePizarro3** [comment]
One thing I was thinking after #105: now, `xc_functionals` is a sub-section of `DFT`, but with the new `contributions` concept, I think this should be restructured, i.e., delete `xc_functionals` sub-section while keeping the class but inheriting from `BaseModelMethod` instead of `ArchiveSection`.

Does this makes sense for you?

**Comment by JosePizarro3** [comment]
It is also part of the Assignments of the tutorial: https://fairmat-nfdi.github.io/fairmat-tutorial-14-computational-plugins/nomad_simulations/#modelmethod Assignment 2.3

**Comment by JFRudzinski** [comment]
@EBB2675 Can you clarify why this is not covered by the above referenced PRs?

**Comment by EBB2675** [comment]
Yes, Libxc can provide default exact-exchange information for many hybrid functionals. For global hybrids it exposes xc_hyb_exx_coef, and for CAM/range-separated hybrids it exposes xc_hyb_cam_coef with omega, alpha, and beta. However, this does not mean that consuming codes actually use those defaults as-is.

In practice, it is very common to modify the default exact-exchange fraction in simulations. For hybrid functionals, the HF-related setup usually has to be specified explicitly on the code side and is not automatically determined just because the functional is taken from Libxc, as can be seen for example in CP2k

That is why, in the Libxc canonicalization, I did not assign default values for these exchange fractions on the schema side. In the end, this information still needs to be parsed from the code output or input, because the actual value used in the calculation may differ from the Libxc default

and that's why I did not resolve this issue

**Comment by EBB2675** [comment]
an example from VASP:
https://vasp.at/wiki/LIBXC1_Pn

even if we provide some defaults somewhere, taken from libxc or somewhere else, the authoritative source is the OUTCAR

**Comment by ndaelman-hu** [comment]
> That is why, in the Libxc canonicalization, I did not assign default values for these exchange fractions on the schema side. In the end, this information still needs to be parsed from the code output or input, because the actual value used in the calculation may differ from the Libxc default

I agree with this reasoning. As you laid out, there may be a discrepancy between the parameters from LibXC and the code. The story can actually get even far more complicated.

Filling in LibXC can just be a fallback (though it may be imperfect). The code's parameters (when extracted) should indeed always take precedence. I believe this is how we've handled it in other parts of the schema too, right?

I can take up this issue. No prob.

**Comment by EBB2675** [comment]
> I can take up this issue. No prob.

sure, you have self-assigned yourself on Mar 2, 2024 

please let me know if there is anything you need from my side 🚀
